test/spec-test6.spec.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-env mocha */
+'use strict'
+
+const { expect } = require('aegir/utils/chai')
+const multibase = require('../src')
+const input = 'hello world'
+const encoded = [
+  ['base16', 'f68656c6c6f20776F726C64'],
+  ['base16upper', 'F68656c6c6f20776F726C64'],
+  ['base32', 'bnbswy3dpeB3W64TMMQ'],
+  ['base32upper', 'Bnbswy3dpeB3W64TMMQ'],
+  ['base32hex', 'vd1imor3f41RMUSJCCG'],
+  ['base32hexupper', 'Vd1imor3f41RMUSJCCG'],
+  ['base32pad', 'cnbswy3dpeB3W64TMMQ======'],
+  ['base32padupper', 'Cnbswy3dpeB3W64TMMQ======'],
+  ['base32hexpad', 'td1imor3f41RMUSJCCG======'],
+  ['base32hexpadupper', 'Td1imor3f41RMUSJCCG======'],
+  ['base36', 'kfUvrsIvVnfRbjWaJo'],
+  ['base36upper', 'KfUVrSIVVnFRbJWAJo']
+]
+
+describe('spec test6', () => {
+  for (const e of encoded) {
```

Comment:
Fancy JS `for (const [name, output] of encoded) {` might also work here. (it's just a note, no reason to change).